dist/immutable-nonambient.d.ts

```diff
@@ -1928,7 +1940,7 @@
      * so if `notSetValue` is not provided and this method returns `undefined`,
      * that does not guarantee the key was not found.
      */
-    get(key: K, notSetValue?: V): V;
+    get(key: K, notSetValue?: V): V | undefined;
```

Comment:
I would like to suggest we change this to the following:

```
get(key: K): V | undefined;
get(key: K, notSetValue: V): V;
```

If you've provided a `notSetValue`, this should only ever return `undefined` if `V : undefined`. The current iteration therefore demands unnecessary null-handling, if I'm not mistaken.